### DISCUSSION TITLE
revoque GitHub isn't implemented

### DISCUSSION THREAD

**Discussion by opaniagu** [frankie567/httpx-oauth#305] [Q&A]
Why isn't revoke token() implemented for github?

https://docs.github.com/en/rest/apps/oauth-applications?apiVersion=2022-11-28#delete-an-app-authorization

**Comment by frankie567** [comment]
Because it's poorly documented and wasn't aware it was possible with their API. They do not seem to follow the standard OAuth2 protocol for this, so this would require a specific implementation.
